src/DynamoCoreWpf/Properties/Resources.en-US.resx

```diff
@@ -2354,4 +2350,15 @@ Uninstall the following packages: {0}?</value>
   <data name="MessageSamePackageDiffVersInStdLib" xml:space="preserve">
     <value>A different version of the package {1} is already installed as part of the Standard Library. {0} cannot override packages in the Standard Library location.</value>
   </data>
-</root>
+  <data name="PackagePathViewModel_Standard_Library" xml:space="preserve">
+    <value>Standard Library</value>
+  </data>
+  <data name="PackagePathViewSummary2" xml:space="preserve">
+    <value>The default install location is the first path that is not the Standard Library location.</value>
+    <comment>Package path management dialog content that describes about the first non standard libary entry being the default save location.</comment>
```

Comment:
How about saying `Package path management dialog content that displays the first non-standard library entry being the default save location.`